collections/drop_last_while.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Returns all elements in the given collection until the last element that does not match the given predicate
+ *
+ * Example:
+ * ```ts
+ * import { dropLastWhile } from "./drop_last_while.ts";
+ * import { assertEquals } from "../testing/asserts.ts";
+ *
+ * const values = [22, 30, 44];
+ *
+ * const notFourtyFour = dropLastWhile(values, i => i != 44);
+ *
+ * assertEquals(
+ *   notFourtyFour,
+ *   [22, 30]
+ * );
+ * ```
```

Comment:
```suggestion
 * Drop all elements in the given collection until the last element that does not match the given predicate
 *
 * Example:
 * ```ts
 * import { dropLastWhile } from "./drop_last_while.ts";
 * import { assertEquals } from "../testing/asserts.ts";
 *
 * const numbers = [22, 30, 44];
 *
 * const dropLastWhileNumbers = dropLastWhile(numbers, i => i != 44);
 *
 * assertEquals(
 *   dropLastWhileNumbers,
 *   [22, 30]
 * );
 * ```
```

Reply:
I disagree, `notFourtyFour` is more descriptive and generally makes more sense than `dropLastWhileNumbers`